packages/travix-ui-kit/components/slidingPanel/slidingPanel.js

```diff
@@ -132,6 +133,62 @@ export default class SlidingPanel extends Component {
     const panelClassName = classnames(getClassNamesWithMods('ui-sliding-panel', panelMods), className);
     const overlayClassName = getClassNamesWithMods('ui-sliding-panel-overlay', overlayMods);
 
+    let titleContent = title ? (
+      <SlidingPanelHeader
+        leftBlock={leftBlock}
+        onBackButtonClick={onBackButtonClick}
+        rightBlock={rightBlock}
+        useDefaultLeftBlock={useDefaultLeftBlock}
+      >
+        {title}
+      </SlidingPanelHeader>
+    ) : null;
+
+    let subheaderContent = subheader ? (
```

Comment:
After the migration, this will be going to die since subheader property will be deprecated.